docs/en/observability/whats-new.asciidoc

```diff
@@ -3,15 +3,118 @@
 
 Here are the highlights of what's new and improved in {minor-version}.
 
-coming[8.0.0]
-
-// Other versions:
-// {observability-guide-all}/8.1/whats-new.html[8.1] |
-// {observability-guide-all}/8.0/whats-new.html[8.0] |
-// {observability-guide-all}/7.17/whats-new.html[7.17] |
+Other versions:
+{observability-guide-all}/8.3/whats-new.html[8.3] |
+{observability-guide-all}/8.2/whats-new.html[8.2] |
+{observability-guide-all}/8.1/whats-new.html[8.1] |
+{observability-guide-all}/8.0/whats-new.html[8.0] |
+{observability-guide-all}/7.17/whats-new.html[7.17] |
 
 // tag::whats-new[]
 
 // What's new content goes in here. Don't uncomment or remove the tags surrounding this content :)
 
+[discrete]
+== New Synthetics App enhancements for public beta
+
+As we progress towards delivering an all new Synthetic Monitoring capability
+within Elastic Observability, we’re excited to announce two big new enhancements
+for our 8.4 stack release:
+
+- Private testing locations
+- Open access to the managed testing service for all Elastic Cloud users
+
+With private testing locations you can now configure a testing location on
+self-managed infrastructure. This means that you can test applications that
+aren’t publically available on the internet or from locations where we may not
+yet have a point of presence.
+
+A big part of our new Synthetics app is the ability to run synthetic monitoring
+tests on our new global network of managed testing locations. With the 8.4
+release, we’ll no longer be requiring users to request access. If you want to
+use our managed testing infrastructure, simply open the {uptime-app} and click
+*Monitor Management*. There is a fair usage cap in place but all test executions
+are free. We’d love to get you involved and get your feedback.
```

Comment:
Might be worth telling users how to submit feedback.

Reply:
I did think similar...will need to get something concrete via Drew though, might be a late addition

Reply:
Yah definitely not worth delaying the merge. Just something to add later.